Updates to the Starter Kit Guide (#6220)
umbraco/UmbracoDocs: 14/umbraco-cms/tutorials/starter-kit/lessons/3-add-open-graph/step-4.md

1. Go to the **Settings** section.

> 2. Click on **Partial Views** and select **Create...** > **New empty partial view**

Reviewer: ```suggestion
2. Click on **Partial Views** and select **Create...** > **New empty partial view**.
3. **Enter a Name** for the partial view. Let's call it: _OpenGraph_
```

Reviewer: You might have to adjust the below numbering 😀

3. Add the standard view model: `@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage`

>     * We are using compositions and only render this view on pages where the composition exists, which means we need to be a little more specific.

Reviewer: ```suggestion
    * We only render this view on pages where the composition exists, so we need to be more specific.
```
